dashboard/test/ui/step_definitions/steps.rb

```diff
@@ -28,6 +28,18 @@ def replace_hostname(url)
   url.gsub(/(\w+)\.(\w+)\.code\.org/,'\1-\2.code.org')
 end
 
+def color_for_status(status)
+  {
+    submitted: 'rgb(118, 101, 160)',
```

Comment:
if these do map to scss, it might be nice to put the color name in a comment next to each line so that if someone does a global find (and possibly replace) they seem them